tests/create_postgres.nim

```diff
@@ -1,22 +0,0 @@
-import strutils, sequtils, db_postgres
```

Comment:
Why remove this file? It seems useful.

Reply:
This file is used to initialize and clean the postgresql database before and after each test.  It is no longer needed, because the task 'initcommonpostgre' in config.nims completely replaces it.